### ISSUE TITLE
error: Could not bind on given addresses: No error

### ISSUE THREAD

**Issue by Mangaclub** [getdnsapi/stubby#24]
Greetings,

I Installed this programm with the msi installer and executed it like shown below

C:\Windows\System32>"C:\Program Files\Stubby\stubby.exe" -C "C:\Program Files\Stubby\stubby.yml" -l
[08:03:38.669718] STUBBY: Read config from file C:\Program Files\Stubby\stubby.yml
error: Could not bind on given addresses: No error

glad to see that the error was no error ;D

**Comment by saradickinson** [comment]
Yes, nice error message... Could you let me know what version of Windows you are using? 

Try to run the command from a 'Administrator' command prompt window - I think that should fix it.

**Comment by wtoorop** [comment]
Yes looking where the error comes from (line 781 in stubby.c), I think the error message should report the value returned by `getdns_context_set_listen_addresses` and also we should have a platform independent `perror` like `_getdns_perror` in `platform.[ch]`

**Comment by WinLinKer** [comment]
![image](https://user-images.githubusercontent.com/3109626/33198541-a0e73778-d126-11e7-9c7b-652f6af25dd5.png)
